src/components/feedbacks/FeedbackItem.vue

```diff
@@ -1,28 +1,24 @@
 <template>
   <BaseBox>
     <article>
-      <p>{{}}</p>
+      <p v-if="roadmap">{{ feedback.status }}</p>
       <!-- status - activated in the Roadmap, hidden in the FeedbackList -->
       <div class="feedback__votes">
         <ArrowUp />
-        <p>{{}}</p>
-        <!-- upvotes -->
+        <p>{{ feedback.upvotes }}</p>
       </div>
       <div class="feedback__content">
-        <h2>{{}}</h2>
-        <!-- title -->
-        <p>{{}}</p>
-        <!-- content -->
-        <BaseButton>{{}}</BaseButton>
-        <!-- category -->
+        <h2>{{ feedback.title }}</h2>
+        <p>{{ feedback.description }}</p>
+        <BaseButton>{{ feedback.category }}</BaseButton>
```

Comment:
I guess you should do sth with this button, but it might be done later?

Reply:
I made a semantic mistake here - this is only a like-a-button component (any button functionality, only it is styled like a button). I added props likeAButton as an alternative in the BaseButton component and put this prop here. Is that solution correct? 

In the next branches, I've already added props text to the BaseButton and I replaced interpolation with this prop.

Reply:
props likeAButton and the div + p combination removed from BaseButton, BaseButton in FeedbackItem replaced with div + p